group_vars/i_am_a_y-a-m-l_file

```diff
@@ -0,0 +1,2 @@
+---
+this_file_has_problems: TRUE
```

Comment:
This file should be picked up by yamllint and annotated with a failure, but yamllint is not actually checking it.